ci/releases.yml

```diff
@@ -48,7 +48,9 @@ suite-desktop github release:
       - codesign
   when: manual
   script:
-    - release-it --ci --config=ci/release-it.yml --disable-metrics
+    - gh config set prompt disabled
```

Comment:
This turns off the "interactivness"?

Reply:
Yes, by default, it ignores all the options and will still ask you to enter them interactively. This will turn it off.